tests/tox.ini

```diff
@@ -35,8 +35,10 @@ commands =
 description = Run integration tests
 deps =
     -r {tox_root}/requirements-test.txt
+allowlist_externals =
+    sudo
 commands =
-    pytest -v \
+    sudo -E {envpython} -m pytest -v \
```

Comment:
why running sudo is needed?

Reply:
we now use ``ensure_image_contains_paths_bare`` in the sanity test, which checks the image layers instead. Because of this, we need sufficient permissions to check the ``/var/lib/docker`` folder.